src/sections/reference.html

```diff
@@ -1,7 +1,7 @@
 <div class="h-full bg-[url('../../img/Rectangle-3.webp')] bg-cover bg-center">
   <div>
     <ul
-      class="py-48 pr-10 text-right font-outfit text-xl md:pr-32 md:text-2xl lg:pr-40"
+      class="pt-36 pb-64 pr-10 text-right font-outfit font-normal text-xl md:pr-32 md:text-2xl lg:py-60 lg:pr-40"
```

Comment:
this is how the figma is 

![refrence figma](https://github.com/DevRamona/the_Baumesiter/assets/116556392/7bc1db39-4813-4999-9105-60d3f2836b10)

that is how the padding looks like when the page is scrolled 

![the actual page](https://github.com/DevRamona/the_Baumesiter/assets/116556392/c718c32d-ce9f-44e0-b06a-8cf30c4a1deb)

Reply:
Resolved check it out.